HTML/contact.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE HTML>
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" type="text/css" href="styles/main.css" media="screen"/>
+</head>
+
+<body>
+  <div class="site-wrap">
+              <nav class="nav-banner">
+                  <div class="menu-item">
+                      <div class="nav-text">
+                          <h3><a href="index.html"> Home </a> </h3>
+                      </div>
+                  </div>
+
+                  <div class="menu-item">
+                      <div class="nav-text">
+                        <h3><a href="about.html">About</a></h3>
+                      </div>
+                  </div>
+
+                  <div class="menu-item">
+                      <div class="nav-text">
+                          <h3><a href="contact.html"> Contact </a> </h3>
+                      </div>
+                  </div>
+              </nav>
+
+
+        <div class="contact-wrap">
+                <section class="side side-a">
+                    <h1 class="center-header"> Congratulations on choosing to improve your business! </h1>
+                    <img src="svgs/partycomputer.svg" alt="computer"/>
+                    <div class="center-header">
+                        <h1> Email: Jfalzarano888@gmail.com </h1>
+                    </div>
+                    <div class="line"></div>
+                </section>
+
+                <section class="side side-b">
+                  <h1 class="center-header"> How would you like to get this <br> party
+                    started?  </h1>
+                  <img src="svgs/partyphone.svg" alt="computer"/>
+                  <div class="center-header">
+                      <h1> Phone: 908-210-2566 </h1>
+                  </div>
+                  <div class="line"></div>
+                </section>
+        </div>
+        <div class="thanks-box-center">
+          <div class="logo-wrap">
+              <img src="svgs/logo.svg" alt="computer"/>
```

Comment:
for relative paths, always start then with './'. so `svgs/logo.svg` becomes `./svgs/logo.svg`